test/unit/mocked_data/test_get_interfaces/normal/_show__interface_ethernet1_3__interface___show_.xml

```diff
@@ -3,9 +3,9 @@
     <name>ethernet1/3</name>
     <duplex>full</duplex>
     <type>0</type>
-    <state_c>auto</state_c>
+    <state_c>down</state_c>
```

Comment:
What is state_c versus state (i.e. which one is the operational state versus being administratively down)?

Reply:
`state` is op, `state_c` is admin